db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Product.destroy_all
+Location.destroy_all
+
+10.times do
+  FactoryGirl.create(:product)
+  FactoryGirl.create(:location)
```

Comment:
@tmikeschu or maybe @ethanbennett @DavidKnott Do we need any locations to be seeded or will those be created each time a user adds a checkpoint to a path (or starts a path)?  I don't know the answer - looking for feedback/guidance.